Library/ViewModels/PledgeDisclaimerViewModel.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import KsApi
+import ReactiveSwift
+
+public protocol PledgeDisclaimerViewModelOutputs {
+  var presentTrustAndSafety: Signal<Void, Never> { get }
+}
+
+public protocol PledgeDisclaimerViewModelInputs {
+  func learnMoreTapped()
+}
+
+public protocol PledgeDisclaimerViewModelType {
+  var inputs: PledgeDisclaimerViewModelInputs { get }
+  var outputs: PledgeDisclaimerViewModelOutputs { get }
+}
+
+public final class PledgeDisclaimerViewModel: PledgeDisclaimerViewModelType,
+  PledgeDisclaimerViewModelInputs, PledgeDisclaimerViewModelOutputs {
+  public init() {
+    self.presentTrustAndSafety = self.learnMoreTappedProperty.signal
+  }
+
+  private let learnMoreTappedProperty = MutableProperty(())
```

Comment:
Are we using `MutableProperty` or `Signal.pipe`? Asking because I'm starting to see both in the codebase.

Reply:
Yeah both are kind've fine, there are one or two scenarios in which a `MutableProperty` is preferable, `pipe` just saves us from having to `skipNil()` on the property's `signal`. I'd say if unsure, use `MutableProperty<T, Error>`.